File: src/main/java/com/example/ChatServer/MessageSchema.java
package com.example.ChatServer;


import org.springframework.stereotype.Repository;

@Repository
public class MessageSchema {

    private String message;
    private String sendBy;

    public String getSendTo() {
        return sendTo;
    }

    public void setSendTo(String sendTo) {
        this.sendTo = sendTo;
    }

    private String sendTo;
    private String Time;


    public String getTime() {
        return Time;
    }

    public void setTime(String time) {
        Time = time;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getSendBy() {
        return sendBy;
    }

    public void setSendBy(String sendBy) {
        this.sendBy = sendBy;
    }

}

File: lib/db/MessageSchema.dart
import 'package:hive/hive.dart';
part 'MessageSchema.g.dart';
@HiveType(typeId: 0)
class MessageSchema extends HiveObject{
  @HiveField(0)
  final String message;
  @HiveField(1)
  final String sendBy;
  @HiveField(2)
  final String sendTo;
  @HiveField(3)
  final String time;

  MessageSchema(this.message, this.sendBy, this.sendTo, this.time);


}
